core/src/main/java/org/hisp/dhis/android/core/validation/ValidationRule.java

```diff
@@ -68,22 +67,22 @@ public abstract class ValidationRule extends BaseNameableObject implements CoreO
     @ColumnAdapter(LeftValidationRuleExpressionColumnAdapter.class)
     public abstract ValidationRuleExpression leftSide();
 
-    abstract String leftSideExpression();
-
-    abstract String leftSideDescription();
-
-    @ColumnAdapter(MissingValueStrategyColumnAdapter.class)
-    abstract MissingValueStrategy leftSideMissingValueStrategy();
+//    abstract String leftSideExpression();
+//
+//    abstract String leftSideDescription();
+//
+//    @ColumnAdapter(MissingValueStrategyColumnAdapter.class)
```

Comment:
Remove these comments.
You could probably remove the class `MissingValueStrategyColumnAdapter` as well.